__init__.py

```
@@ -55,8 +66,8 @@
                 is_app = True
                 if os.path.isdir(path):
                     continue
-                with open(path) as f:
-                    for l in f.read().split("\n"):
+                with open(path) as fi:
+                    for l in fi.read().split("\n"):
```

Ruff

Ambiguous variable name: `l`

(E741)